### ISSUE TITLE
'manta get configuration' does not work on 1.22.3

### ISSUE THREAD

**Issue by mcaubet** [eth-cscs/manta#28]
```bash
(ansible2.9) 💻 [mcaubet@castaneda:~]# manta get configuration
2024-02-13 12:52:20 | INFO  | /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/mesa-0.27.3/src/common/authentication.rs:47 — Reading CSM authentication token from configuration file
2024-02-13 12:52:20 | INFO  | /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/mesa-0.27.3/src/common/authentication.rs:140 — Validate Shasta token against https://api.cmn.alps.cscs.ch/apis/cfs/healthz
2024-02-13 12:52:20 | INFO  | /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/mesa-0.27.3/src/common/authentication.rs:151 — Shasta token is valid
thread 'main' panicked at src/common/cfs_configuration_utils.rs:76:43:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

(ansible2.9) 💻 [mcaubet@castaneda:~]# manta --version
manta 1.22.3
```

**Comment by Masber** [comment]
Hi Marc, 

this is a bug because manta is trying to print a commit id which does not exists, this is weird because normally CFS will assign a commit id to every CFS layer automatically, otherwise, the cluster creation process would fail. This is something done by CFS, manta just submits the job to create the CFS session.

I have fixed this so manta will show a "Not defined" text whenever the commit is is missing.

thank you

**Comment by Masber** [comment]
solved in 1.22.4
